logging/troubleshooting/cluster-logging-alerts.adoc

```diff
@@ -20,7 +20,13 @@ include::modules/cluster-logging-collector-alerts-viewing.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* For more information on the Alerting UI, see xref:../../monitoring/managing-alerts.html#managing-alerts[Managing alerts].
+* For more information on the Alerting UI, see 
+ifdef::openshift-enterprise,openshift-origin[]
+xref:../../monitoring/managing-alerts.html#managing-alerts[Managing alerts.]
```

Comment:
This was already in place, but it might be part of the errors. Swap .html for .adoc: 
xref:../../monitoring/managing-alerts.adoc#managing-alerts[Managing alerts.]